### PR TITLE
Input table manifest must exist, if it is specified by the flag.

### PR DESCRIPTION
**Pull request by michaljurecko**
Slack: https://keboolaglobal.slack.com/archives/C055WKPHYSD/p1699884466075089

Changes:
- CLI: Added validation: input table manifest must exist, if it is specified by the flag.

### CODE REVIEW

File: internal/pkg/slicer/slicer.go [hunk 1]
```diff
@@ -20,12 +20,13 @@ import (
...
-	OutPath         string `validate:"required" json:"outPath" mapstructure:"table-output-path"`
-	OutManifestPath string `validate:"required" json:"outManifestPath" mapstructure:"table-output-manifest-path"`
+	config.Config        `json:"config" mapstructure:",squash"`
+	Name                 string `validate:"required" json:"name"  mapstructure:"table-name"`
+	InPath               string `validate:"required"  json:"inPath" mapstructure:"table-input-path"`
+	InManifestPath       string `json:"inManifestPath"  mapstructure:"table-input-manifest-path"`
+	InManifestMustExists bool   `json:"-" mapstructure:"-"` // true in CLI, false in processor
+	OutPath              string `validate:"required" json:"outPath" mapstructure:"table-output-path"`
+	OutManifestPath      string `validate:"required" json:"outManifestPath" mapstructure:"table-output-manifest-path"`
 }
```
Comment (michaljurecko) [inline internal/pkg/slicer/slicer.go:30]: Pridany field `InManifestMustExists bool`.

File: internal/pkg/cli/cli.go [hunk 2]
```diff
@@ -42,6 +42,10 @@ func Run(logger log.Logger) error {
 		defer pprof.StopCPUProfile()
 	}
 
+	// In CLI, the manifest must exist, if it is specified by the flag.
+	// In processor, the manifest is autodetect, so it may not exist.
+	cfg.InManifestMustExists = true
+
 	// Slice table
```
Comment (michaljurecko) [inline internal/pkg/cli/cli.go:49]: V procesore ma definicia kazdej najdenej tabulky vyplnenu cestu ku manifestu.
Az neskor sa rozlisuje, ci subor existuje alebo nie (nemusi existovat).

V CLI ale chceme aby subor existoval, ak bol zadany, tak pridavam tento switch.

File: test/cli/manifest-not-found/args [hunk 3]
```diff
@@ -0,0 +1,5 @@
+--table-name mytable
+--table-input-path $IN_DIR/table.csv
+--table-input-manifest-path $IN_DIR/table.csv.manifest
```
Comment (michaljurecko) [inline test/cli/manifest-not-found/args:3]: Pridany E2E test.

File: test/cli/manifest-not-found/expected-stderr [hunk 4]
```diff
@@ -0,0 +1 @@
+Error: manifest "*/table.csv.manifest" not found
```
Comment (michaljurecko) [inline test/cli/manifest-not-found/expected-stderr:1]: Chyba z E2E testu.

**Review by ErikZigo — APPROVED** [review]
go je pro me 🇪🇸 vesnice, ale letmym pohledem na test a zmeny si rikam ze by to melo delat co popisujes :)

**Pull request merged**